src/app/wrappers/expand-input-wrapper/ux-expand-input-ng1.directive.ts

```diff
@@ -0,0 +1,22 @@
+angular.module('app').directive('uxExpandInputNg1', function() {
```

Comment:
When I click on the search icon and then click cancel, then click the search icon again, the search box doesnt show until I click the search icon another time